README.md

```diff
@@ -7,6 +7,29 @@ This script is used to calculate the ICV value from `subj-dir`/`mri`/`talairach.
 * Python module [Fire](https://github.com/google/python-fire)
 
 ## Usage
-Default:
-`calculate_icv.py` --> will run on current directory, assumes `Cerebel_vols.csv` is in the same directory
+NAME
+    `calculate_icv.py`
+    
+SYNOPSIS
+   `calculate_icv.py <flags>`
 
+DESCRIPTION
+    We assume 
+    1. Executable xfm2det is under root dir
+    2. output file Cerebel_vols.csv is under root dir
+
+FLAGS
+```
+    --root=ROOT
+        Default: '/f...
```

Comment:
this will be the current working directory from os.getcwd()

Reply:
Updated